app/components/custom_list/option_list_row/option_list_row.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {intlShape} from 'react-intl';
+import {
+    Text,
+    View,
+} from 'react-native';
+import {makeStyleSheetFromTheme} from 'app/utils/theme';
+
+import CustomListRow from 'app/components/custom_list/custom_list_row';
+
+export default class OptionListRow extends React.PureComponent {
+    static propTypes = {
+        id: PropTypes.string,
+        theme: PropTypes.object.isRequired,
+        ...CustomListRow.propTypes,
+    };
+
+    static contextTypes = {
+        intl: intlShape,
+    };
+
+    onPress = () => {
+        if (this.props.onPress) {
+            this.props.onPress(this.props.id, this.props.item);
+        }
+    };
+
+    render() {
+        const {
+            enabled,
+            selectable,
+            selected,
+            theme,
+            item,
+        } = this.props;
+
+        const {text, value} = item;
+        const style = getStyleFromTheme(theme);
+
+        return (
+            <CustomListRow
+                id={value}
+                theme={theme}
+                onPress={this.onPress}
+                enabled={enabled}
+                selectable={selectable}
+                selected={selected}
+            >
+                <View style={style.textContainer}>
+                    <View>
```

Comment:
Do we need this inner `View`?

Reply:
I couldn't get the positioning to work without it